experimental/entity/entity.proto

```diff
@@ -0,0 +1,198 @@
+syntax = "proto3";
+package entity;
+
+option go_package = "./;entity";
+
+
+message EntityMessage {
+    // UID may contain / to indicate parent folder's
+    string UID = 1; 
+
+    // dash, ds, alert, folder, svg, png, df, dqr, ... (will validate body)
+    string kind = 2; 
+
+    // Metadata managed by the underlying storage engine(s)
+    StorageMetadata meta = 3; 
+
```

Comment:
Do we no longer need the `properties: map<string, string>` ?

Reply:
All user configured data would live in the payload (name,descr,tags, etc) -- all the data that is derived from storage system will live in `meta`

The properties map may exist -- but it is embedded in the payload bytes

Reply:
So we are expecting the `payload` to be a combination of `Envelope` (https://github.com/grafana/grafana-plugin-sdk-go/pull/527/files/bd226be67f511d35b133f87f17f8d60596173132#diff-7be6e2fececb62de1c26961cfb095307d304ab9abac09d22c4814bcb2bb6a2fcR4) + body? I would interpret the comment `the raw entity bytes` as just the body, aka entity json without envelope

Reply:
as structured here... it is the envelope+body

With "RawFileEntity" -- ie images etc -- the envelope will need to be augmented from the path or elewhere in storage